spec/models/green_lanes/categorisation_spec.rb

```diff
@@ -104,6 +104,45 @@
     end
   end
 
+  # rubocop:disable RSpec/AnyInstance
+  describe '.load_from_s3' do
+    let(:json_string) do
+      '[{
+          "category": "1",
+          "regulation_id": "D0000001",
+          "measure_type_id": "400",
+          "geographical_area": "1000",
+          "document_codes": [],
+          "additional_codes": []
+        }]'
+    end
+
+    context 'when the file exists in S3' do
+      subject(:s3_categories) { described_class.load_from_s3 }
+
+      # rubocop:disable RSpec/MessageChain
```

Comment:
So I personally don't mind message chains. Should we just add this to the rubocop config file?

Reply:
Here I tried to avoid message chains and allow any instance by mocking `Rails.application.config.persistence_bucket`, but it is not possible as it is a frozen object. In other places probably you can avoid message chain by using mock object. So I not sure if we need to disable this cop globally. Having said that, I am not sure what is the issue of using message chain either.

Reply:
Yeh, I'd agree @willfish  we could just disable that rubocop rule globally?

Reply:
Yeah, I also can't see how that object is frozen in the initialiser but probably I'm missing something and it's not super important anyway

Reply:
OK, I'll move it to global level then.